fast_api_als/database/db_helper.py

```diff
@@ -16,6 +16,12 @@
     write a commong function that logs this response code with appropriate context data
 """
 
+logging.basicConfig(filename = 'logs.log' , level = logging.INFO , format = '%(asctime)s : %(filename)s : %(lineno)d :: %(levelno)s : %(message)s')
+
+def HTTPStatusCode_Log(res) : 
+    logging.debug('HTTPStatusCode : {}'.format(res['ResponseMetadata']['HTTPStatusCode']))
```

Comment:
This when you'll be reading logs and you see `HTTPStatusCode: 200`, will you able to figure out what does it mean, which function operation had 200 as response. whose response is 200??

Reply:
I tried to provide context in the caller function of HTTPStatusCode_Log() itself. Like for example, in line 48 of db_helper.py.